.changeset/great-candies-win.md

```diff
@@ -0,0 +1,9 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
```

Comment:
Shouldn't this be a breaking change (therefore a major?)

Reply:
I don't think we're supposed to change those - they are auto-generated and [don't mean what you'd think they mean](https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/dev/wiki/sync/changesets).

Reply:
Dear me 😑